libp2p/pubsub/floodsub.py

```diff
@@ -104,3 +94,26 @@ def remove_peer(self, peer_id):
         It is invoked after the unsubscription announcement.
         :param topic: topic to leave
         """
+
+    def _get_peers_to_send(
+            self,
+            topic_ids: Iterable[str],
+            src: ID,
+            origin: ID) -> Iterable[ID]:
+        """
+        Get the eligible peers to send the data to.
+        :param src: the peer id of the peer who forwards the message to me.
+        :param origin: the peer id of the peer who originally broadcast the message.
+        :return: a generator of the peer ids who we send data to.
+        """
+        for topic in topic_ids:
+            if topic not in self.pubsub.peer_topics:
+                continue
+            for peer_id_str in self.pubsub.peer_topics[topic]:
+                peer_id = id_b58_decode(peer_id_str)
+                if peer_id in (src, origin):
+                    continue
+                # FIXME: Should change `self.pubsub.peers` to Dict[PeerID, ...]
+                if str(peer_id) not in self.pubsub.peers:
```

Comment:
Combine them into
```python
if peer_id in (src, origin) or peer_id_str not in self.pubsub.peers:
```

Reply:
Personally, I think it is more clear to separate them into two `if`. I'd prefer to stick to the original style if we are not against this style strongly.